interfaces/automated-testing/geo-awareness/geo-awareness.yaml

```diff
@@ -147,57 +147,100 @@ components:
       required:
         - uomDimensions
         - verticalReferenceType
-        - verticalReference
-        - longitude
-        - latitude
       properties:
         uomDimensions:
           $ref: '#/components/schemas/UomDimensions'
         verticalReferenceType:
           $ref: '#/components/schemas/VerticalReferenceType'
         verticalReference:
+          description: >-
+            Height above vertical reference datum, in units of uomDimensions.
           type: number
+          default: 0
         longitude:
+          description: >-
+            Longitude, degrees east of prime meridian.
           type: number
+          default: 0
         latitude:
+          description: >-
+            Latitude, degrees north of the equator.
           type: number
+          default: 0
 
-    UASZonesCheckRequest:
+    GeozonesCheckRequest:
       type: object
       required:
-        - position
-        - uSpaceClass
-        - acceptableRestrictions
-        - startDateTime
-        - endDateTime
+        - checks
       properties:
-        position:
-          $ref: '#/components/schemas/Position'
-        uSpaceClass:
-          $ref: '#/components/schemas/USpaceClass'
-        acceptableRestrictions:
+        checks:
           type: array
           items:
-            $ref: '#/components/schemas/Restriction'
-        startDateTime:
+            $ref: '#/components/schemas/GeozonesCheck'
+
+    GeozonesCheck:
+      type: object
+      required:
+        - filterSets
+      properties:
+        filterSets:
+          description: >-
+            Select Geozones which match any of the specified filter sets.
+          type: array
+          items:
+            $ref: '#/components/schemas/GeozonesFilterSet'
+
+    GeozonesFilterSet:
+      description: >-
+        Set of filters to select only a subset of Geozones.  Only Geozones which are applicable to all specified filters within this filter set should be selected.
+      type: object
+      properties:
+        position:
+          description: >-
+            If specified, only select Geozones encompassing this position.
+          anyOf:
+            - $ref: '#/components/schemas/Position'
+        after:
```

Comment:
I think this is clearer than `startDateTime` because nothing starts at this time -- we are interested in some Geozones that start before this time, and some Geozones that start after this time.  The important thing is that we only look for Geozones that some temporal part that extends past this time.